### PR TITLE
Default user model location changed to support Laravel >=8

### PR DESCRIPTION
**Pull request by upi20**
Now default user model location `App\Models\User`, still you can override this config by `user-activity.php`

### CODE REVIEW

**Comment by desod-vn** [comment]
you just update UserInstance in config model as well.

File: src/Listeners/LockoutListener.php [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 namespace Haruncpi\LaravelUserActivity\Listeners;
 
-use App\User;
+use App\Models\User;
```
Comment (haruncpi) [inline src/Listeners/LockoutListener.php]: Please remove this import statement completely.

File: src/Models/Log.php [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Log extends Model
     public $dates = ['log_date'];
```
Comment (haruncpi) [inline src/Models/Log.php]: Remove this one

File: src/Models/Log.php [hunk 3]
```diff
@@ -17,7 +17,7 @@ public function __construct() {
 
     public function getDateHumanizeAttribute()
     {
-        return $this->log_date->diffForHumans();
+        return \Carbon\Carbon::parse($this->attributes['log_date'])->diffForHumans();
```
Comment (haruncpi) [inline src/Models/Log.php]: Here use `Carbon::parse` and import it at top `use Carbon\Carbon;`

**Pull request merged**